src/services/YotiSessionCompletionProcessor.ts

```diff
@@ -398,4 +411,22 @@ export class YotiSessionCompletionProcessor {
 		  });
 	  }
   }
+
+  async sendErrorMessageToIPVCore(f2fSession: ISessionItem, errorMessage: string) : Promise<any> {
+  	try {
+  		await this.f2fService.sendToIPVCore({
+  			sub: f2fSession.subject,
+  			state: f2fSession.state,
+  			error: errorMessage,
```

Comment:
I think the ticket says that `error` should always be `access_denied`, or has something changed?

Reply:
oh Yes, i saw that but thought could be custom. Thanks. I will add this custom errorMsg to the errorDescription instead.